### PR TITLE
Add Faraday 0.9.0 compatiblity

### PR DESCRIPTION
**Pull request by alexchee**
I think I updated the middleware for Faraday 0.9.0, the specs pass, but I haven't used Faraday so I'm not sure if this is the only change needed.

### CODE REVIEW

**Comment by jherdman** [comment]
Neat! I'll take at this change shortly.

File: zeppelin.gemspec [hunk 1]
```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
 
   s.rubyforge_project = 'zeppelin'
 
-  s.add_dependency 'faraday', '< 0.9.0'
+  s.add_dependency 'faraday'
```
Comment (jherdman) [inline zeppelin.gemspec:18]: Could you add a version constraint like this: `< 0.10.0`? I'd rather not see a completely open-ended dependency.

Comment (alexchee) [inline zeppelin.gemspec:18]: Good idea

**Comment by jherdman** [comment]
Awesome! Can you flatten this down into one commit? We're good to go after that.

**Comment by alexchee** [comment]
I think I squashed it into 1 commit, never tried it before.

**Comment by jherdman** [comment]
Thanks! Well done.

**Pull request merged**